Adjudicate substantive changes to the powerbi baseline markdown following the new format

amart241 opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

Changed formatting and content of PowerBI baseline to match new formatting structure
<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

## 💭 Motivation and context ##

Consolidated format and clearer to read and understand layout
<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such -->
<!-- as "closes" or "resolves" to auto-close them on merge. -->

## 🧪 Testing ##

N/A
<!-- How did you test your changes? How could someone else test this PR? -->
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->

<!--
## 📷 Screenshots (if appropriate) ##

Uncomment this section if a screenshot is needed.

-->

## ✅ Pre-approval checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - *eschew scope creep!*
- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- These boxes should remain unchecked until the pull request has been -->
<!-- approved. -->

## ✅ Post-merge checklist ##

<!-- Remove any of the following that do not apply. -->

ahuynhMITRE commented on baselines/powerbi.md:
```diff
@@ -137,9 +83,9 @@ security groups, instead of allowing the entire agency to publish data
-The Publish to Web feature SHOULD be disabled unless the agency mission requires the capability.
-- _Rationale:_ TODO
+#### MS.POWERBI.1.1v1
+Disable the Publish to Web feature unless the agency mission requires the capability.
```
"shall" and "should" language is still expected in the policies. Only removed from the previous "policy groups".

ahuynhMITRE replied:
left this comment only for the first policy to not create duplicate comments

ahuynhMITRE reviewed (changes requested):
agree with David's comments and main comment from me was re-adding the "shall" and "should" language  into each of the policies.

ahuynhMITRE commented:
some comments: 

1. Update text following each policy group to the  match the guide below:

This section helps reduce security risks related to [Policy Group]. This includes [Scope of policies ] (This sentence is as needed).
Example: "This section helps reduce security risks related to the sharing of files with users external to the agency. This includes guest users, users who use a verification code and users who access an Anyone link."

2. remove links to admin centers and re-add them as a generic step. This was recently voted on as the current way forward by by the team.

amart241 commented:
I have removed a total of 3 policies from the Power BI baseline. I most recently removed the logging related policy because it is inconsistent across the new "fabric" and current Power BI, it also requires and Azure subscription. #315 were the other 2 policies that I removed.

ahuynhMITRE commented:
> This comment is tied to the text after each policy group. See policy group 1 below:

"1. Publish to Web
Power BI has a capability to publish reports and content to the web. This capability creates a publicly accessible web URL that does not require authentication or status as an AAD user to view it. While this may be needed for a specific use case or collaboration scenario, it is a best practice to keep this setting off by default to prevent unintended and potentially sensitive data exposure.

If it is deemed necessary to make an exception and enable the feature, admins should limit the ability to publish to the web to only specific security groups, instead of allowing the entire agency to publish data to the web."

This would be updated to the following:
"1. Publish to Web

This section helps reduce security risks related to Power Bi's capability to publish to the web. "

This was a note by Ted since a lot of the text will be covered by the rationale and is inconsistent across baselines.

ahuynhMITRE commented:
> 

good point, will add as a parking lot item for tomorrow. Probably lost in the many conversations / reviews over the past months and open to whatever works best.

ahuynhMITRE reviewed (approved):
all good after the discussion at the stand up!

nanda-katikaneni commented:
@amart241 @buidav - seems there are conflicts for this branch to merge into emerald, can you please update/rebase and resolve conflicts.
